jsonnet/kube-prometheus/components/alertmanager.libsonnet

```diff
@@ -189,4 +189,91 @@ function(params) {
       },
     },
   },
+
+  clusterRole: {
+    apiVersion: 'rbac.authorization.k8s.io/v1',
+    kind: 'ClusterRole',
```

Comment:
I don't believe this needs to be a cluster role, does it? We have everything setup with plain `Role`s

Reply:
Some components already have `ClusterRole` instead of `Role`. Should I create a new Role for those? Or just add the psp to the existing ClusterRole?

Reply:
I had the chance to try out this commit today and I couldn't deploy anything that used `Role` instead of `ClusterRole` . I'm not sure if I messed up the `Role`s somehow or if PSPs are supposed to work with `ClusterRole` only 😬

Reply:
That's odd, I'm certain it works with Roles because that's how we run this.